pkgs/development/ocaml-modules/kicadsch/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildDunePackage
+, fetchFromGitHub
+, ocaml
+}:
+
+assert (lib.versionAtLeast ocaml.version "4.09");
+
+buildDunePackage rec {
+  pname = "kicadsch";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "jnavila";
+    repo = pname;
```

Comment:
I'm not finding this repository.

https://github.com/jnavila/kicadsch

Does not exist for me.

Reply:
Also notice `plotkicadsch` has .tbz release file:

https://github.com/jnavila/plotkicadsch/releases/download/v0.9.0/plotkicadsch-0.9.0.tbz

(This is useful in fixing version in META file)